data/src/main/kotlin/com/vultisig/wallet/data/models/payload/BlockChainSpecfic.kt

```diff
@@ -40,6 +40,7 @@ sealed class BlockChainSpecific {
         val priorityFee: BigInteger,
         val fromAddressPubKey: String? = null,
         val toAddressPubKey: String? = null,
+        val tokenAccountExists: Boolean? = null,
```

Comment:
This will not work , as this change is not in the protobuf , thus it doesn't get passed to pair device
and this should caues keysign failure , because they will not agree on what they are signing

Reply:
Actually, I updated the "common data" submodule.

https://github.com/vultisig/commondata/pull/16
![Screenshot_20241025_092434_Chrome](https://github.com/user-attachments/assets/3d84b083-75fa-4799-bb01-84f8edb4501e)

Reply:
There is no need to add new stuff to commondata now ,  please refer to how IOS handle it 
https://github.com/vultisig/vultisig-ios/blob/ton-stake-function/VultisigApp/VultisigApp/Chains/Solana.swift#L44

Reply:
> This will not work , as this change is not in the protobuf , thus it doesn't get passed to pair device and this should caues keysign failure , because they will not agree on what they are signing

The current Android codes is the same as ios codes
https://github.com/vultisig/vultisig-android/blob/6a7dd2a0977cf78bf47fe76125789e731ebe5a0e/data/src/main/kotlin/com/vultisig/wallet/data/chains/helpers/SolanaHelper.kt#L63-L69
https://github.com/vultisig/vultisig-ios/blob/77da3dbec92dfbe77fc36242cbf7cf0b1f7fb151/VultisigApp/VultisigApp/Chains/Solana.swift#L46-L54

Reply:
@johnnyluo The current IOS  does not covert the problem